bin/gltf-pipeline.js

```diff
@@ -78,6 +79,7 @@ fs.readFile(gltfPath, function (err, data) {
 
         combineMeshes(gltf);
         combinePrimitives(gltf);
+        combineNodes(gltf);
```

Comment:
I was thinking of calling `combineNodes` first, but right now it works based off the fact that each node has a combined mesh. Is it okay if I call `combineMeshes` at the end again to recombine the new meshes?